docs/attack-techniques/AWS/aws.exfiltration.ec2-security-group-open-port-22-ingress.md

```diff
@@ -32,4 +32,13 @@ Opens ingress traffic on port 22 from the Internet (0.0.0.0/0).
 
 ```bash title="Detonate with Stratus Red Team"
 stratus detonate aws.exfiltration.ec2-security-group-open-port-22-ingress
-```
\ No newline at end of file
+```
+## Detection
+
+
+You can use the CloudTrail event <code>AuthorizeSecurityGroupIngress</code> when:
+
+- <code>requestParameters.cidrIp</code> is <code>0.0.0.0/0</code> (or an unknown external IP)
+- and <code>requestParameters.fromPort</code>/<code>requestParameters.toPort</code> is not a commonly exposed port or corresponds to a known administrative protocol such as SSH or RDP
```

Comment:
Should we mention here that this is also reported by AWS Trusted Advisor as well as AWS Inspector?

Reply:
Good idea! I think it's worth it, but I need to investigate if/how they detect other TTPs as well. Adding to the backlog and merging this already